tests/integrated/test-drift-instability/2fluid.cxx

```diff
@@ -292,6 +294,8 @@ int physics_init(bool restarting) {
   dump.add(rho_s, "rho_s", 0);
   dump.add(wci,   "wci", 0);
 
+  ylow=mesh->StaggerGrids?CELL_YLOW:CELL_CENTRE;
```

Comment:
Some whitespace please! Making code easy to read for the next person is really important. I know I bang on about clang-format a lot, but it really means you don't need to think about formatting -- let the computer do it for you.

Also, instead of `ylow`, consider something a bit more descriptive: `cell_location` for instance. Naming things is hard, but it's worth spending a bit of time on it.
What does this variable really represent? It's value is only possibly `CELL_YLOW`, so when we see it used later on, seeing `ylow` might make us think that it is always `CELL_YLOW`.

Reply:
I am not certain about `cell_location` - as some cells are always going to be at `CELL_CENTRE`
It is not worse then before - `CELL_YLOW` implied even stronger to be at `CELL_YLOW`.

`maybe_ylow`? `someone_tried_to_add_stagger_support_but_did_not_finish`? `maybe_in_the_future_ylow`?
If you want the code to be clean, it should be removed ...